_CoqProject

```diff
@@ -11,4 +11,6 @@ src/cvc4Solver.ml
 src/polyaSolver.ml
 src/smtTactic.mllib
 
+theories/Coqlib.v
```

Comment:
This file seems to be from compcert. It can't be in this repository. If this pull request depends on it, then we need to find a different way to factor the pull request.